### PR TITLE
Strickinato changes

### PR DESCRIPTION
**Pull request by strickinato**
@avh4 @tesk9 @raorao 

I took some time to review curriculum.

1. Amazing. Love the app. Great descriptions. Nicely done!

2. I've added a couple things that address some questions that I thought I would have if I was a student. git commits are very descriptive and are 1 change at a time, so it should be reasonable to review.

<3

### CODE REVIEW

File: Our First Full Feature.md [hunk 1]
```diff
@@ -57,22 +57,25 @@ This is a **union type declaration**. It defines a new type that we can use in o
...
-    "meow"
-  else if pet == Dog
-    "woof"
-  else
-    "..silence.."
+  case pet of
+    Cat ->
+      "meow"
+    Dog ->
+      "woof"
```
Comment (raorao) [inline Our First Full Feature.md]: My use of an if statement was intentional. i wanted students to realize they could use values in "normal" implementation code, like a conditional. Case statements can look alien, if you aren't familiar with the syntax, and you could easily assume that the left hand side of a case has to be a type, not a value. 

Students might also think that you *must* use a case statement with a union type, which is of course mistaken.

File: Our First Full Feature.md [hunk 2]
```diff
@@ -189,7 +192,7 @@ Now that `Model` can describe the current translation direction, we need to chan
 
 Inside `Update.update`, we know two things – the current `Msg` we received, as well as the current state of the `model`, which includes its current `direction`. That's all the information we need – if the current `model.direction` is `EmojiToText`, `Update.update` should return a `model` with a `direction` value of `TextToEmoji`, and vice versa.
 
-When working with custom types, case expressions are helpful in writing conditional logic. In our case, the code would look something like this:
+Once again, we can use our friend `case ___ of` to handle the different conditions. In our case, the code would look something like this:
```
Comment (raorao) [inline Our First Full Feature.md]: ❤️❤️❤️

File: The Elm Architecture.md [hunk 3]
```diff
@@ -178,4 +178,19 @@ This is an example of a **union type** declaration in Elm. We'll go further into
 
 Elm doesn't force us to use a union type called `Msg` for `messages` — we could just as easily model `messages` as strings, numbers, or records. It is convention, however, to model data using union types whenever appropriate. Don't worry if you are somewhat lost — union types are one of the more difficult concepts to learn as a beginner to Elm, and it's fine for now if they are simply magic words you know you can recite.
 
+To review one more time, a Html.App.beginnerProgram needs:
+  * An initial version of our model (`model : Model`)
```
Comment (raorao) [inline The Elm Architecture.md]: Love this!

**Comment by avh4** [comment]
I'm fixing this up right now..

**Comment by avh4** [comment]
nevermind, @strickinato  is fixing :)

**Comment by strickinato** [comment]
1. Removed values commit
2. Changed type signature and gave elm formatting to Elm Architecture Final Reiteration

**Comment by strickinato** [comment]
Oops - leftout 1 thing

**Comment by strickinato** [comment]
👍  << Ready to merge I think!

**Pull request merged**